src/config/constants.js

```diff
@@ -57,6 +57,13 @@ export const MAIL_BREAK_LINE = '%0D%0A';
 export const HEADER_LOGO_HEIGHT = 40;
 export const HEADER_HEIGHT = 65;
 
+export const LEFT_MENU_WIDTH = 240;
+
+// todo: use environment variables
+export const PICTURE_BASE_URL = 'https://graasp.eu';
```

Comment:
You can get rid of that constant, and everything related to it, I think it's not even used.